hawkrest/__init__.py

```diff
@@ -61,7 +86,7 @@ def authenticate(self, request):
 
         try:
             receiver = Receiver(
-                lookup_function,
+                lookup_credentials,
```

Comment:
I think this code would be easier to read if you just made it `lambda id: self._credentials_lookup(id)`